cypress/e2e/save-nametag-spec.cy.ts

```diff
@@ -1,6 +1,6 @@
 describe("Save nametag button", () => {
   it("is able to save nametag and persist it across logins", () => {
-    const email = "foobar@example.com";
+    const email = `foobar@example.com`;
```

Comment:
Why do we only change the quotation marks here?

Reply:
Right, that's not related to this change. I've backed it out now.